### PR TITLE
feature: add proxy name field to graphene obj

### PR DESCRIPTION
**Pull request by fregataa**
Add `proxy` and `name` field to StorageProxy graphene object

**Checklist:** (if applicable)

- [x] Milestone metadata specifying the target backport version
- [x] API server-client counterparts (e.g., manager API -> client SDK)

### CODE REVIEW

**Review by agatha197 — APPROVED** [review]
LGTM

File: src/ai/backend/manager/models/storage.py [hunk 1]
```diff
@@ -199,14 +200,18 @@ class StorageVolume(graphene.ObjectType):
     class Meta:
         interfaces = (Item,)
 
-    # id: {proxy_name}:{name}
+    id = graphene.String()  # "{proxy}:{name}""
```
Comment (agatha197) [inline src/ai/backend/manager/models/storage.py]: @fregataa Please revert this line.
It occurs this error.
image

**Review by agatha197 — CHANGES_REQUESTED** [review]

File: src/ai/backend/manager/models/storage.py (hunk 1 above)
Comment (fregataa) [reply src/ai/backend/manager/models/storage.py]: I reverted this. but I am curious why it raises such error..

Comment (agatha197) [reply src/ai/backend/manager/models/storage.py]: I think it's TypeError. `Interface field Item.id expects type ID but StorageVolume.id is type String.`

**Review by agatha197 — APPROVED** [review]
LGTM

File: src/ai/backend/manager/models/storage.py [hunk 2]
```diff
@@ -199,14 +200,18 @@ class StorageVolume(graphene.ObjectType):
...
     fsprefix = graphene.String()
     path = graphene.String()
     capabilities = graphene.List(graphene.String)
     hardware_metadata = graphene.JSONString()
     performance_metric = graphene.JSONString()
     usage = graphene.JSONString()
+    proxy = graphene.String(
+        description=description_msg("24.03.1", "Name of the proxy which this volume belongs to.")
+    )
+    name = graphene.String(description=description_msg("24.03.1", "Name of the storage."))
```
Comment (achimnol) [inline src/ai/backend/manager/models/storage.py]: Shouldn't the introduced version be `24.03.0`?

**Review by achimnol — CHANGES_REQUESTED** [review]

File: src/ai/backend/manager/models/storage.py (hunk 2 above)
Comment (fregataa) [reply src/ai/backend/manager/models/storage.py]: I updated it. thank you!

**Review by achimnol — APPROVED** [review]

**Pull request merged**